### DISCUSSION TITLE
Bessere Übersetzung - Better translation

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#173] [Q&A] [answered]
Some things are a bit clumsy translated into German, for example: 

"Setze Start"                >>>    beser wäre:" Startpunkt"
"Setze Ziel"                  >>>                     "Zielpunkt"
"Hinzufügen Start"      >>>                      "Route erweiten(Start)"
"Hinzufügen Ziel"        >>>                      "Route erweitern(Ziel)"

not that important, it works that way...

**Comment by devemux86** [comment]
@Leeb64 thanks for the translations!

I renamed the extend action to "Add" to differentiate it from "Set".
- "Set end": set or move destination
- "Add end": always add a new destination

Or do you think the "Extend route (direction)" is a better name?

---

Please everyone, when you see things that seem strange in a language, let me know.

**Reply by Leeb64** [reply]
"verlängern" is not wrong,but,I think "erweitern"is better in such a case.
"Ende setzen" is strange. "Ziel setzen" or "Zielpunkt setzen" is much better.
"Ende hinzufügen"is also strange "Route erweitern(->Ziel)" would be better understandable(I think)..
Also the other direktion:"Hinzufügen Start" ...... somewhat problematic....I think "Route erweitern(->Start) would be better.

**Comment by devemux86** [comment]
I will rename the extend route actions to:
- "Extend route (start)", "Route erweitern (Start)"
- "Extend route (end)", "Route erweitern (Ziel)"

What about the "Set start" /  "Set end"?
K1 / K2 have: "Setze als Start" / "Setze als Ziel".

**Reply by Leeb64** [reply]
"Setze als Start" and "Setze als Ziel" is OK!
Everyone would understand this.

**Comment by devemux86** [comment, marked as answer]
The improved translations are available in [Cruiser 3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

@Leeb64 thanks for your help!

**Comment by Leeb64** [comment]
Next problem with the translation: Valhalla has an option "use living streets" in German its translated "Lebendige Strassen benutzen".
???? what the hell are "living streets"????
"Lebendige Strassen" means streets that are alive....
Are this streets with a lot of traffic ?

**Reply by devemux86** [reply]
@Leeb64 thanks for exploring all settings, Valhalla has extensive options!

There is a description in [Valhalla website](https://valhalla.openstreetmap.de/):

_"This value indicates the willingness to take living streets. This is a range of values between 0 and 1. Values near 0 attempt to avoid living streets and values near 1 will favor living streets. The default value is 0 for trucks, 0.1 for cars, buses, motor scooters and motorcycles. Note that sometimes living streets are required to complete a route so values of 0 are not guaranteed to avoid living streets entirely."_

**Reply by Leeb64** [reply]
The more correct translation would probably be: "Anwohnerstrassen".
"Lebende Strassen" actually doesn't exist...

**Reply by devemux86** [reply]
> The more correct translation would probably be: "Anwohnerstrassen".

It is "Use residential streets"?
Indeed seems better, I will update it.

**Comment by bios4** [comment]
> Please everyone, when you see things that seem strange in a language, let me know.

Some suggestions for updated German translations (based on Cruiser 3.0.41.b8).
Most of them are to be aligned to other translations throughout the app or at least to be consistent to other translations on their respective screens.

"Korrigieren Sie falsch platzierte Wegpunkte" --> "Falsch platzierte Wegpunkte korrigieren" (Settings > Routing > Options > Advanced) (BRouter)
"Tankstelle" --> "Tankstellen" (Settings > Navigation > Display > nearby POIs)
"Ladestation" --> "Ladestationen" (Settings > Navigation > Display > nearby POIs)
"Ändern Sie die Reihenfolge der Panels" --> "Reihenfolge der Paneele anpassen" (Settings > Navigation > Display as well as Settings > Application > Display)
"Folge der Position, nachdem das Ziel erreicht wurde" --> "Der aktuellen Position folgen, nachdem das Ziel erreicht wurde" (Settings > Navigation > Advanced)
"Zeige die Benachrichtigungen in der Navigation" --> "Benachrichtigungen während der Navigation anzeigen" (Settings > Navigation > Advanced)
"Benachrichtige auch über einen zweiten, eng aufeinanderfolgenden nahen Richtungswechsel" --> "Zusammengefasst über einen knapp folgenden Richtungswechsel benachrichtigen" (Settings > Voice guidance > Advanced)
"Zeige Track auf der Karte" --> "Track auf der Karte anzeigen" (Settings > GPS recording)
"Zeige Standort an der Pfeilspitze anstatt in der Pfeilmitte" --> "Standort an der Pfeilspitze anstatt in der Kreismitte anzeigen" (Settings > Location)
"Reduziere GPS-Fehler" --> "GPS-Fehler reduzieren" (Settings > Location > Advanced)
"Verwende die Lautstärketasten für den Kartenzoom" --> "Lautstärketasten für den Kartenzoom verwenden" (Settings > Map)
"Zeige auf OpenStreetMap/Google Maps/Mapillary" --> "Auf OpenStreetMap/Google Maps/Mapillary anzeigen" (long press on map > more)
"Prüfe verfügbare Routingdaten" --> "Verfügbare Routingdaten prüfen" (long press on map > more)

For the sake of completeness the tile descriptions "Setze als Start/Zwischenziel/Shaping-Point/Ziel" (long press on map) should also be renamed to "Als Start/Zwischenziel/Shaping-Point/Ziel setzen" to match the rest of the translations (i.e. put the verbs at the end).

**Reply by devemux86** [reply]
@bios4 thanks for the translations, I updated the texts.

Except for POI, we currently use the singular to describe them, as their text is used in several places.
Or we should maintain singular + plural in all languages.

**Comment by Leeb64** [comment]
While we're at it...
What sounds strange to me:
"Als Start setzen,Als Ziel setzen,Als Shaping Point setzen,Als Zwischenziel setzen"
It would be just as good or better to simply: “Start(punkt) setzen,Ziel(punkt) setzen,Zwischenziel setzen,Shaping Point setzen”.
The “Als” is completly unnecessary.

**Reply by r11d11** [reply]
When I saw the "Als" I  immediately thought the same as Leeb64. The "Als" is completely unnecessary.

**Reply by devemux86** [reply]
Thanks, I will update the texts.

**Reply by bios4** [reply]
Thanks for the update(s) Emux!

**Reply by bios4** [reply]
Unfortunately I missed two translations @devemux86, maybe you could update them too for a future version:

"Setze Wegpunkte auf Straße" --> "Wegpunkte auf Straße setzen" (long press route menu)
"Zeige Berge" --> "Geländerelief anzeigen" (Settings > Map > Layers) (translation based on the term "hillshading" used in English)

**Reply by devemux86** [reply]
@bios4 thanks, I will update the texts.

**Comment by bios4** [comment]
With the new features and menus in 3.0.43 (B14) I'd like to suggest the following translation changes for the German UI @devemux86:

1) "Abschnitt-Neuberechnung" --> "Abschnitt neu berechnen"
2) "Routen-Neuberechnung" --> "Route neu berechnen"
3) "Füllen mit Shaping-Points" --> "Mit Shaping-Points füllen"

The first one might possibly fix the ugly line break, all three suggestions make the entries fit to the rest of the translations (with the verbs at the end).

<img src="https://github.com/devemux86/cruiser/assets/145460147/5d955ee8-cd93-4098-a1af-4c8c0839153e" width="33%">
<img src="https://github.com/devemux86/cruiser/assets/145460147/4faeb271-dcea-443c-8ebb-3593db3967f7" width="33%">

**Reply by devemux86** [reply]
@bios4 thanks, I will update the texts.

**Comment by bios4** [comment]
The German translation in Main > Settings > Map > Appearance: "Markergruppen (Zoom)" is not really self-explaining.

My suggestion would be to update this to "Symbole gruppieren ab Zoomstufe" (oder alternativ "Marker gruppieren ab Zoomstufe").

**Reply by devemux86** [reply]
@bios4 thanks, actually it is the zoom level where the marker grouping ends.

**Reply by devemux86** [reply]
We will change the text of all languages.

**Comment by bios4** [comment]
Shouldn't "Stop GPS recording" be better renamed to "Pause GPS recording" ("GPS-Aufzeichnung pausieren")?
As the icon represents the "pause" sign and recording can be resumed by tapping "Start GPS recording" again, it would make sense to me.

<img src="https://github.com/devemux86/cruiser/assets/145460147/ccaf3b51-2f03-4315-b99d-6a655db48f70" width="25%">

**Reply by devemux86** [reply]
GPS recording creates a new segment with each start, so the recorded tracks are not continuous between stops.
So essentially is not a real pause, as it does not connect the old track with the new recorded one.

Regarding the icon, we could probably change it?
(I thought the 2-lines icon was more "readable")

**Reply by 0709wiwiwi** [reply]
A recorded gpx (trk) track with multiple (trkseg) tracksegments (due to pause or interrupitions) is usually considered to be one single record track.  The Pause Icon by 2 lines icons is fine.

**Reply by bios4** [reply]
When it is like described (and the final result can be exported to ONE file) the icon is fine - but then the naming should be "Pause", too. 😉

**Reply by devemux86** [reply]
Or we could change both icons:
Activate|Deactivate
:-:|:-:
![on](https://github.com/devemux86/cruiser/assets/3484020/f788722f-8e80-42ea-a311-99463f9eaa52)|![off](https://github.com/devemux86/cruiser/assets/3484020/24244a56-00f4-4268-9c59-20481c59badf)
